src/Core/src/Platform/Android/ContentViewGroup.cs

```diff
@@ -42,6 +41,21 @@ public ContentViewGroup(Context context, IAttributeSet attrs, int defStyleAttr,
 			_context = context;
 		}
 
+		public ICrossPlatformLayout? CrossPlatformLayout
+		{
+			get; set;
```

Comment:
I'm not sure about this but I think the platform view should never have a strong reference to the virtual view.
I think the strong references should flow like this:
VirtualView/VirtualView's Properties (eg. ItemSource) --> Handler/Manager/PlatformView

Handler/Manager --> PlatformView/Native Listeners

but whenever you need the opposite direction it should be a weak reference. It's meaningless to have a platform view that the user have already lost the reference to its VirtualView/Handler.

Reply:
I don't know how to post a comment without it being a review 😅. Feel free to dismiss this one.

Reply:
That's the point of this PR, to make that weak reference possible. The old implementation was implicitly keeping a strong reference through the internal CrossPlatformMeasure/CrossPlatformArrange Funcs. This way, we have the option of making the reference a weak one (and still allow handlers to modify the CPM/CPA implementations). 

The iOS implementation in this PR already uses a weak reference for this. Since we don't currently have leaks on Windows/Android, I've left this as a strong reference to avoid the extra overhead of the weak reference target check. So we _could_ change those to weak references in the future, if we need to. 

That said, I _could_ be convinced to make _all 3_ platforms use weak references just for the symmetry, assuming the performance impact of the weak reference checks is small. 

@jonathanpeppers @PureWeen @mattleibow any thoughts on that?

Reply:
The above property is Android only, so there isn't an issue there? Really only need to be concerned about cycles on iOS/Catalyst.